lib/shopify-cli/project.rb

```diff
@@ -95,6 +95,11 @@ def project_name
         File.basename(current.directory)
       end
 
+      def clear
```

Comment:
This is a useful method. We may want to enhance the `current` method to call `clear` if `force_reload` is set rather than passing it down as a parameter - that would keep our behaviour more consistent across the methods.